recipes/pyhash/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "pyhash" %}
+{% set version = "0.9.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyhash-{{ version }}.tar.gz
+  sha256: cff5c81d613163fc59d623d4546d9be55b46ecd0e573b59057b1bb112a497763
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - cpuid
```

```suggestion
    - cpuid  # [x86]
```
https://github.com/flier/pyfasthash/blob/20a53f9bb7bf15f98e3e549f523b49e1e0f62e15/setup.py#L272